### PR TITLE
interstitial centering support

### PR DESCRIPTION
**Pull request by can-soykarafakili**

### CODE REVIEW

File: PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m [hunk 1]
```diff
@@ -94,6 +94,7 @@ - (void)viewWillAppear:(BOOL)animated {
     [HyBidLogger debugLogFromClass:NSStringFromClass([self class]) fromMethod:NSStringFromSelector(_cmd) withMessage:[NSString stringWithFormat:@"%@ %@", [self.class description], NSStringFromSelector(_cmd)]];
 
     isStatusBarHidden = [[UIApplication sharedApplication] isStatusBarHidden];
+    self.view.backgroundColor = [UIColor blackColor];
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m:97]: Was the color supposed to be black or white? We did it white on Android but I can't remember exactly what PMs wanted.

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m:97]: @eros902002 I made it black based on this ticket: https://pubnativegmbh.atlassian.net/browse/OMI-457

Comment (eros902002) [reply PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m:97]: Cool. Sorry. My bad then.

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m:97]: No worries. I forgot what did i have for the lunch most of the time. 😂

**Pull request merged**